concent_api/core/payments/bankster.py

```diff
@@ -310,7 +310,7 @@ def settle_overdue_acceptances(
             requestor_eth_address=requestor_ethereum_address,
             provider_eth_address=provider_ethereum_address,
             min_block_timestamp=oldest_payments_ts,
-            transaction_type=TransactionType.FORCE,
+            transaction_type=TransactionType.SETTLEMENT,
```

Comment:
@dybi
`FORCE` used to refer to **forced payments** (i.e. the settlement payments). I suggested renaming it because it's confusing but it has not been done yet. Now @rwrzesien is renaming `FORCE` to `SETTLEMENT` and making `FORCE` return **forced subtask payments** instead just like I suggested back then.